notebooks/Intro_To_Mast_Aladin_Lite.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        display(mast_aladin)
</pre><p>I think these examples of positioning the sidecar widget are great.  I am less sure about how well a single mast-aladin instance will behave once it's been displayed in multiple viewports.  Until we study that more, maybe it would be safer to create a new mast-aladin instance for each of the new sidecar views.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/mast-aladin-lite/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>